### PR TITLE
sdio_probe: set voltage use CMD5

### PR DESCRIPTION
**Pull request by yangsong8-ai**
*Note: Please adhere to [Contributing Guidelines](https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md).*

### CODE REVIEW

**Comment by nuttxpr** [comment]
[**\[Experimental Bot, please feedback here\]**](https://github.com/search?q=repo%3Aapache%2Fnuttx+13552&type=issues)



I need the content of the Pull Request to assess if it meets the NuttX requirements. Please provide the following:

* **Link to the Pull Request:**  This is essential so I can see the proposed changes.
* **Description of the changes:**  If the PR description doesn't fully explain the context, purpose, and implementation details, please provide them. 

Once I have this information, I can effectively determine if the PR meets the NuttX requirements.

File: drivers/mmcsd/sdio.c [hunk 1]
```diff
@@ -407,6 +408,25 @@ int sdio_probe(FAR struct sdio_dev_s *dev)
   /* Receive R4 response */
 
   ret = SDIO_RECVR4(dev, SDIO_CMD5, &data);
+
```
Comment (acassis) [inline drivers/mmcsd/sdio.c:411]: remove empty line

File: drivers/mmcsd/sdio.c [hunk 2]
```diff
@@ -407,6 +408,25 @@ int sdio_probe(FAR struct sdio_dev_s *dev)
...
+  if (ret != OK)
+    {
+      goto err;
+    }
+
+  bit = ffs(data);
```
Comment (acassis) [inline drivers/mmcsd/sdio.c:417]: Please include comment before this block

File: drivers/mmcsd/sdio.c [hunk 3]
```diff
@@ -407,6 +408,25 @@ int sdio_probe(FAR struct sdio_dev_s *dev)
...
+  if (bit)
+    {
+      bit -= 1;
+      data &= 3 << bit;
+    }
+  else
+    {
+      ret = ERROR;
```
Comment (xiaoxiang781216) [inline drivers/mmcsd/sdio.c:425]:
```suggestion
      ret = -EINVAL;
```

File: drivers/mmcsd/sdio.c (hunk 1 above)
Comment (yangsong8-ai) [reply drivers/mmcsd/sdio.c]: done

File: drivers/mmcsd/sdio.c (hunk 2 above)
Comment (yangsong8-ai) [reply drivers/mmcsd/sdio.c:418]: done

File: drivers/mmcsd/sdio.c (hunk 3 above)
Comment (yangsong8-ai) [reply drivers/mmcsd/sdio.c]: done

**Review by GUIDINGLI — APPROVED** [review]

**Pull request merged**